vmdb/app/models/vim_performance_analysis.rb

```diff
@@ -486,14 +486,14 @@ def self.find_child_perf_for_time_period(obj, interval_name, options = {})
     klass, _   = Metric::Helper.class_and_association_for_interval_name(interval_name)
 
     user_cond = nil
-    user_cond = klass.send(:sanitize_sql_for_conditions, options[:conditions]) if options[:conditions]
-    cond =  klass.send(:sanitize_sql_for_conditions, ["(timestamp > ? AND timestamp <= ?)", start_time.utc, end_time.utc])
-    cond += klass.send(:sanitize_sql_for_conditions, [" AND capture_interval_name = ?", interval_name]) unless interval_name == "daily"
+    user_cond = klass.sanitize_sql_for_conditions(options[:conditions]) if options[:conditions]
+    cond =  klass.sanitize_sql_for_conditions(["(timestamp > ? AND timestamp <= ?)", start_time.utc, end_time.utc])
+    cond += klass.sanitize_sql_for_conditions([" AND capture_interval_name = ?", interval_name]) unless interval_name == "daily"
     cond =  "(#{user_cond}) AND (#{cond})" if user_cond
 
     if obj.kind_of?(MiqEnterprise) || obj.kind_of?(MiqRegion)
-      cond1 = klass.send(:sanitize_sql_for_conditions, {:resource_type => "Storage",             :resource_id => obj.storage_ids})
-      cond2 = klass.send(:sanitize_sql_for_conditions, {:resource_type => "ExtManagementSystem", :resource_id => obj.ext_management_system_ids})
+      cond1 = klass.sanitize_sql_for_conditions({:resource_type => "Storage",             :resource_id => obj.storage_ids})
```

Comment:
@tenderlove does `sanitize_sql_for_conditions` need to be routed via send?  Is this something we need to rework for upcoming Rails 4 upgrade?